### DISCUSSION TITLE
【求助】关于算法备案的问题

### DISCUSSION THREAD

**Discussion by bh4ffu** [zai-org/ChatGLM3#1234] [Q&A] [answered]
关于算法备案，需要提供【互联网信息服务算法安全自评估报告】，请教以下两个问题：
1、开源的大模型私有化部署需要做算法备案，官方是否支持备案，是否存在法律或者道义上的问题？
2、算法备案需要提供算法的技术细节，有哪些渠道可以获取到？

thanks!

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
开源模型只提供了模型权重，允许商用，但是开源模型不提供算法服务备案的任何工作，望理解

**Reply by bh4ffu** [reply]
感谢回复！
另外，chatglm系列的模型有技术报告可以参考么？
